src/popup/routes.tsx

```diff
@@ -1,23 +1,30 @@
 import * as React from 'react'
 import { MENU } from '~/constants/menu'
 import BottomNavigation from '~/components/BottomNavigation'
-import { Route, Routes } from 'react-router-dom'
+import { Route, Routes as RRRoutes } from 'react-router-dom'
```

Comment:
any reason to rename it to `RRRoutes`? What does it means?

Reply:
RRRoutes, not the best name, but it should mean `Routes` from react-router.

The file name is called `Routes` that's why